contents/tutorials/nextjs-cookie-banner.md

```diff
@@ -119,41 +128,161 @@ This creates an ugly but functional cookie banner at the bottom of our site you
 
```

Comment:
I cant comment on the line above for some reason, but change the sentence to:

`This creates an ugly but functional cookie banner at the bottom of our site. You can customize and style it how you like.`